docs/news/2020-07-06-Dockstore-1.9.0.rst

```diff
@@ -0,0 +1,39 @@
+Dockstore 1.9.0
+===============
+
+Highlighted new features include:
+---------------------------------
+
+-  Language plugin support, allowing for additional support of Galaxy workflows as a preview feature
```

Comment:
Language plugin support is an implementation detail. Even though it's important, it's not a user-facing feature; support for Galaxy is.

tldr; I would just mention Galaxy here, and have a section further below on implementation details.

I guess it depends who we think the audience is.